index.html

```diff
@@ -0,0 +1,699 @@
+    />
+  </head>
+  <body class="flex">
+    <nav class="flex nav-top">
+      <div class="flex nav-top-links">
+        <ul class="flex">
+          <li class="nav-top-link">
+            <a href="#"><strong>News</strong></a>
+          </li>
+          <li class="nav-top-link"><a href="#">Events</a></li>
+          <li class="nav-top-link"><a href="#">Business</a></li>
+          <li class="nav-top-link"><a href="#">AMAs</a></li>
+          <li class="nav-top-link"><a href="#">Spaces</a></li>
+          <li class="nav-top-link"><a href="#">Terms & Conditions</a></li>
+        </ul>
+        <ul class="flex">
+          <li class="nav-top-link"><a href="#">News</a></li>
+          <li class="nav-top-link"><a href="#">Events</a></li>
+          <li class="nav-top-link"><a href="#">Business</a></li>
+          <li class="nav-top-link"><a href="#">AMAs</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <div id="nav-fixed">
+      <ul class="flex nav-bottom-links">
+        <li class="nav-top-link"><a href="#">Latest</a></li>
+        <li class="nav-top-link"><a href="#">Hard Fork</a></li>
+        <li class="nav-top-link"><a href="#">Plugged</a></li>
+        <li class="nav-top-link"><a href="#">ReadMe</a></li>
+        <li class="nav-top-link"><a href="#">Growth Quarters</a></li>
+        <li class="nav-top-link"><a href="#">Shift</a></li>
+        <li class="nav-top-link"><a href="#">Neural</a></li>
+      </ul>
+      <div class="search-logo">
+        <i class="fa fa-search" aria-hidden="true"></i>
+      </div>
+    </div>
+
+    <main>
+      <section class="grid articles">
+        <div class="coverLeft">
+          <a href="#" class="article-1">
+            <div class="flex top-article-1">
+              <img src="images/tnw.svg" alt="The Next Web Logo" />
+              <h4>Thursday - June 24, 2020</h4>
+            </div>
+            <div class="flex bttn-article-1">
+              <h1>
+                How my company succesfully changed my role from on-site to
+                remote.
+              </h1>
+              <p>
+                by Sheraya Kothari<i class="fa fa-clock" aria-hidden="true"></i>
+                1 hour ago<i class="fa fa-share-alt" aria-hidden="true"></i>6
+              </p>
+            </div>
+          </a>
+        </div>
+        <div class="coverRight-1">
+          <a href="#" class="article-2">
+            <div>
+              <h2>
+                Join our 15th annual flagship event for free
+              </h2>
+            </div>
+          </a>
+        </div>
+        <div class="coverRight-2">
+          <a href="#" class="article-3">
+            <h2>
+              How to claim 2 months of free YouTube Premium from T-Mobile
+            </h2>
+            <p>
+              by Rachel Kaser<i class="fa fa-clock" aria-hidden="true"></i>1
+              hour ago<i class="fa fa-share-alt" aria-hidden="true"></i>8
+            </p>
+          </a>
+        </div>
+      </section>
```

Comment:
**Top 3 articles section**

- [ ]  Top 3 articles section contains one bigger image on the leftmost part of the row (around two-thirds of the width) ([link to page](https://thenextweb.com/)). But in your project, it covers much more than the two-thirds of the width. Please kindly fix it.

![image](https://user-images.githubusercontent.com/60184961/91212437-df4fc600-e718-11ea-8fb3-91637f47571b.png)

- [ ] For medium screens, the TNW logo should disappear.

![image](https://user-images.githubusercontent.com/60184961/91212290-ab74a080-e718-11ea-9686-571160e5611b.png)